app/src/main/java/io/multy/api/ApiServiceInterface.java

```diff
@@ -58,6 +59,9 @@
     @GET("api/v1/wallets/{walletIndex}/verbose")
     Call<ResponseBody> getWalletVerboseByIndex(@Path ("walletIndex") int walletIndex);
 
+    @DELETE("api/v1/wallet/{walletIndex}")
```

Comment:
Avoid using rxJava. We don't need it for now. Default Call<Object> is quite much enough for our purposes.